draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2673,11 +2712,25 @@ When hashing passwords using any function described in this document, an adversa
 who learns the output of the hash function (or potentially any intermediate value,
 e.g., the output of hash\_to\_field) may be able to carry out a dictionary attack.
 To mitigate such attacks, it is recommended to first execute a more costly key
-derivation function (e.g., PBKDF2 {{!RFC2898}} or scrypt {{!RFC7914}}) on the password,
+derivation function (e.g., PBKDF2 {{?RFC2898}} or scrypt {{?RFC7914}}) on the password,
 then hash the output of that function to the target elliptic curve.
 For collision resistance, the hash underlying the key derivation function
 should be chosen according to the guidelines listed in {{hashtofield-expand}}.
 
+Constant-time implementations of all functions in this document are STRONGLY
+RECOMMENDED for all uses, to avoid leaking information via side channels.
```

Comment:
Should we reference {{VR20}} again here (or after "for security against timing attacks" below)?